src/Verifier/SAW/Heapster/SAWTranslation.hs

```diff
@@ -632,6 +632,8 @@ instance TransInfo info =>
       return $ error "translate: CharRepr"
     [nuMP| StringRepr UnicodeRepr |] ->
       returnType1 stringTypeOpenTerm
+    [nuMP| StringRepr _ |] ->
+      return $ error "translate: StringRepr non-unicode"
```

Comment:
Do we actually intend/need to handle non-unicode cases?

Reply:
Not at this point. We only need to support what actually shows up in the Crucible we are pointing Heapster at.